.github/workflows/versioned-docs.yml

```
@@ -0,0 +1,133 @@
+    steps:
+      - name: Check out repository with full history
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Full history needed for versioning
+          
+      - name: Setup Conda Environment
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          environment-file: environment.yml
+          activate-environment: edbook
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)